### PR TITLE
split footer component

### PR DESCRIPTION
**Pull request by OksanaKhorushko**

### CODE REVIEW

File: src/components/Copyright/Copyright.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const Copyright = () => {
+  return (
+    <div className="footer__copyright">
```
Comment (Ihor-Kleponosov) [inline src/components/Copyright/Copyright.jsx]: Change class name to copyright

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import './Footer.scss';
-import { Button, Icon, Logo } from '../';
+import { Copyright, FooterMainNavigation, FooterSecondaryNavigation, Logo, Sabscribe, SocialMedia } from '../';
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.jsx]: Please run the linter and check the issues. This string looks too long

File: src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -8,82 +8,17 @@ const Footer = () => {
...
-              </ul>
-            </nav>
-            <div className="sabscribe">
-              <input placeholder="Email address"></input>
-              <Button href="#">
-                Sabscribe
-              </Button>
-            </div>
+            <FooterMainNavigation />
+            <Sabscribe />
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.jsx]: sUbscribe

File: src/components/Sabscribe/Sabscribe.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import './Sabscribe.scss';
+import { Button } from '../';
+// import PropTypes from 'prop-types';
+
+const Sabscribe = () => {
```
Comment (Ihor-Kleponosov) [inline src/components/Sabscribe/Sabscribe.jsx]: The same sUbscribe

File: src/components/Sabscribe/Sabscribe.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div className="sabscribe">
+      <input placeholder="Email address"></input>
+      <Button href="#">
+        Sabscribe
```
Comment (Ihor-Kleponosov) [inline src/components/Sabscribe/Sabscribe.jsx]: The same

File: src/components/Sabscribe/Sabscribe.scss [hunk 6]
```diff
@@ -0,0 +1,15 @@
+@import '../../scss/variables';
+
+.sabscribe {
```
Comment (Ihor-Kleponosov) [inline src/components/Sabscribe/Sabscribe.scss]: class too

**Review by Silwero — APPROVED** [review]

**Pull request merged**